quoridor-logic-server/src/main/java/com/rl/mpquoridor/controllers/GameWebSocket.java

```diff
@@ -0,0 +1,42 @@
+package com.rl.mpquoridor.controllers;
+
+import com.rl.mpquoridor.models.Position;
+import com.rl.mpquoridor.models.actions.TurnAction;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.List;
+
+@Controller
+public class GameWebSocket {
+
+    @Autowired
+    private SimpMessagingTemplate messageSender;
+
+
+    @MessageMapping("/movePawn/{gameId}")
```

Comment:
The order should be opposite.
{gameId}/movePawn

or if its not possible then
/game/{gameId}/movePawn